scripts/data_validation/validate_udw_vs_udp.py

```diff
@@ -71,16 +81,17 @@ def main():
 
     # Set up ENV for both UDW and UDP
     dir_path = os.path.dirname(os.path.realpath(__file__))
-    print(dir_path)
-    print(os.path.abspath(os.path.abspath(os.pardir)))
+    logger.debug(dir_path)
+    logger.debug(os.path.abspath(os.path.abspath(os.pardir)))
     ENV_UDW = get_env_file('/secrets/env.hjson', 'hjson')
     ENV_UDP = get_env_file('/secrets/env_udp.hjson', 'hjson')
-    ENV_CRON_UDW = get_env_file('/secrets/cron.hjson', 'hjson')
-    ENV_CRON_UDP = get_env_file('/secrets/cron_udp.hjson', 'hjson')
+
+    # Use the config files in this project
+    ENV_CRON_UDW = get_env_file('/code/config/cron_udw.hjson', 'hjson')
```

Comment:
This file no longer exists in the latest code.

Reply:
yes. I need to update this accordingly. Doing it now.

Reply:
@jonespm you can check [9739a47](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/1392/commits/9739a47bd43f312781d84082976b20ec8ec5aa08) tomorrow.